modules/client/src/client-common/interfaces/common.ts

```diff
@@ -14,7 +14,7 @@ export enum DaoRole {
  */
 export interface IPluginInstallItem {
   id: string; // ENS domain or address of the plugin's Repo
-  data: Uint8Array;
+  data: string;
```

Comment:
Why aren't we using a uint8 array anymore?

Reply:
Right, but `ethers` should not be taken for granted. It's a helper that could potentially change (hope not). 
The SDK is ethers agnostic, hence `BigNumber`, `BigInteger`, `BigNumberish`, etc are library-specific, but not idiomatic JS to be exported by an SDK.